Workbooks/Azure Monitor - Workspaces/Agent Health/settings.json

```diff
@@ -2,5 +2,6 @@
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
     "name":"Agent Health",
     "author": "Microsoft",
-    "galleries": [{ "type": "workbook", "resourceType": "Azure Monitor", "order": 200 }]
+    "galleries": [],
+	"description": "not shown in any gallery, this workbook is deprecated, please use Workbooks/Azure Monitor - Agents/Overview"
```

Comment:
nit: this looks like it's slightly indented

Reply:
Fixed in latest commit.